apps/verceldashboard/vercel_dashboard.star

```diff
@@ -165,6 +174,12 @@ def get_schema():
                 desc = "Your Vercel API key generated via the Vercel dashboard",
                 icon = "key",
             ),
+            schema.Text(
+                id = CONFIG_TEAM_ID,
+                name = "TEAM ID",
+                desc = "Your Vercel Team ID - required for team deployments. Leave blank for personal deployments.",
```

Comment:
Not sure if you want to put in README or this description but would be good to inform user to ensure the API key has the correct scope for the team as outlined in the API docs

Reply:
Improved description for users

Added additional information to the README including the Configuration